app/src/main/res/values/colors.xml

```diff
@@ -131,37 +131,119 @@
     <color name="fx_mobile_private_layer_color_2" tools:ignore="UnusedResources">@color/photonInk50</color>
     <!-- Search -->
     <color name="fx_mobile_private_layer_color_3">@color/photonInk50</color>
+    <!-- Homescreen background, Toolbar -->
+    <color name="fx_mobile_private_layer_color_4_start" tools:ignore="UnusedResources">@color/photonPurple70</color>
+    <!-- Homescreen background, Toolbar -->
+    <color name="fx_mobile_private_layer_color_4_center" tools:ignore="UnusedResources">@color/photonViolet80</color>
+    <!-- Homescreen background, Toolbar -->
+    <color name="fx_mobile_private_layer_color_4_end" tools:ignore="UnusedResources">@color/photonInk05</color>
+    <!-- App Bar Top (edit), Text Cursor, Selected Tab Check -->
+    <color name="fx_mobile_private_layer_color_accent" tools:ignore="UnusedResources">@color/photonViolet40</color>
+    <!-- Selected tab -->
+    <color name="fx_mobile_private_layer_color_accent_nonopaque" tools:ignore="UnusedResources">@color/photonViolet50A32</color>
+    <color name="fx_mobile_private_layer_color_scrim" tools:ignore="UnusedResources">@color/photonDarkGrey05A45</color>
```

Comment:
is this missing a comment?

Reply:
<img width="618" alt="Screen Shot 2022-02-22 at 4 27 43 PM" src="https://user-images.githubusercontent.com/1190888/155222422-e91e47eb-1fdf-486d-9858-d2ee20ef2359.png">

Nope